.gitignore

```diff
@@ -3,4 +3,6 @@
 /build/
 /out/
 bazel-*
-CMakeSettings.json
\ No newline at end of file
+CMakeSettings.json
+/cmake-build-debug/
```

Comment:
A very minor comment:
I'm not familiar with clion, but I'm guessing there might be a `cmake-build-release` directory if you build a non debug build?
If so, would this be better as `/cmake-build-*/` ?
Would also be nice if this were placed between `/build/` and `/out/`.
Happy if you want to fix this as a follow-up.

Reply:
fixed